To: vim_dev@googlegroups.com
Subject: Patch 7.3.347
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.347
Problem:    When dropping text from a browser on Vim it receives HTML even
	    though "html" is excluded from 'clipboard'. (Andrei Avk)
Solution:   Fix the condition for TARGET_HTML.
Files:	    src/gui_gtk_x11.c


*** ../vim-7.3.346/src/gui_gtk_x11.c	2011-08-10 17:44:41.000000000 +0200
--- src/gui_gtk_x11.c	2011-10-23 20:56:38.000000000 +0200
***************
*** 3081,3087 ****
  
      for (i = 0; i < (int)N_DND_TARGETS; ++i)
      {
! 	if (!clip_html && selection_targets[i].info == TARGET_HTML)
  	    n_targets--;
  	else
  	    targets[j++] = dnd_targets[i];
--- 3081,3087 ----
  
      for (i = 0; i < (int)N_DND_TARGETS; ++i)
      {
! 	if (!clip_html && dnd_targets[i].info == TARGET_HTML)
  	    n_targets--;
  	else
  	    targets[j++] = dnd_targets[i];
*** ../vim-7.3.346/src/version.c	2011-10-20 21:58:20.000000000 +0200
--- src/version.c	2011-10-26 11:35:23.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     347,
  /**/

-- 
I AM THANKFUL...
...for a lawn that needs mowing, windows that need cleaning
and gutters that need fixing because it means I have a home.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
